chromium_src/base/path_service_unittest.cc

```diff
@@ -0,0 +1,15 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+// One of the paths this test attempts to verify on Windows is DIR_ONE_DRIVE.
+// This path doesn't exist on our CIs. To avoid filtering out the entire test
+// let's just skip this path.
+#define BRAVE_PATH_SERVICE_TEST_GET \
```

Comment:
I don't think this should be merged, we can set `DIR_ONE_DRIVE`, we should just create this directory on CI as needed

Reply:
I am checking with DevOps

Reply:
So, @mihaiplesa tried creating the directory and even installing OneDrive - neither helped. It seems like this particular folder ID (FOLDERID_OneDrive) just doesn't get installed with the "server" flavor of Windows OS. We can either patch or disable the test on Windows altogether.

Reply:
I don't want to patch for this so let's just disable it